Model/Currency.php

```diff
@@ -222,6 +235,15 @@ protected function getConsumerIp()
      */
     protected function checkLocalIP($ip)
     {
-        return in_array($ip, ['localhost','127.0.0.1','172.19.0.1','172.25.0.7','172.25.0.1']);
+        $this->_logger->debug('------------- Currency::checkLocalIP() : START ------------------');
+        if ( ! filter_var($ip, FILTER_VALIDATE_IP, FILTER_FLAG_NO_PRIV_RANGE | FILTER_FLAG_NO_RES_RANGE) )
```

Comment:
I don't know where these constants FILTER_VALIDATE_IP, etc are defined. Is that code change missing or did I miss where these are defined?

Reply:
They are PHP constants defined by the run-time.